File: dart/easy/getAlphabetLetterIdx.dart
class GetAlphabetLetterIdx {
  static result(String letter) {
    String alphabet = 'abcdefghijklmnopqrstuvwxyz';
    if (alphabet.contains(letter)) {
      return alphabet.indexOf(letter) + 1;
    }
    return -1;
  }
}

File: js/easy/getAlphabetLetterIdx.js
function getAlphabetLetterIdx(letter) {
  var alphabet = 'abcdefghijklmnopqrstuvwxyz';
  if (alphabet.includes(letter)) {
    return alphabet.indexOf(letter) + 1;
  }
  return -1;
}
console.log(getAlphabetLetterIdx('a'));
console.log(getAlphabetLetterIdx('z'));
console.log(getAlphabetLetterIdx('я'));
